src/core/tests/models/onnx/com.microsoft/gelu.prototxt

```diff
@@ -0,0 +1,46 @@
+ir_version: 3
```

Comment:
Why the oldest `ir_version`? Let's use the newest supported by OV. Same about `opset_import.version`.